src/controllers/hid/hidcontroller.h

```diff
@@ -20,7 +20,7 @@
 class HidController final : public Controller {
     Q_OBJECT
   public:
-    HidController(const hid_device_info deviceInfo);
+    HidController(const hid_device_info deviceInfo, UserSettingsPointer pConfig);
```

Comment:
Please remove the legacy *const-by-value* parameters from all HID controller constructors that have been modified.